libs/design/src/atoms/form/form-field/behaviors/focusable/focusable-interface.ts

```diff
@@ -6,4 +6,4 @@ import { DaffFormFieldControl } from '../../form-field-control';
 export interface DaffFocusable {
 	_control: DaffFormFieldControl;
 	focused: boolean;
-}
\ No newline at end of file
+}
```

Comment:
Wrong PR?

Reply:
The linting of this file was failing build so I just added this fix here